packages/kulala-fmt/package.yaml

```
@@ -0,0 +1,26 @@
+---
+name: kulala-fmt
+description: |
+  kulala-fmt
+  An opinionated .http and .rest files linter and formatter.
+homepage: https://github.com/mistweaverco/kulala-fmt
+licenses:
+  - MIT
+languages:
+  - http
+categories:
+  - Formatter
+  - Linter
+
+source:
+  id: pkg:github/mistweaverco/kulala-fmt@latest
+  asset:
+    - target: darwin_arm64
+      file: kulala-fmt-macos
+    - target: linux_x64
+      file: kulala-fmt-linux
+    - target: win_x64
+      file: kulala-fmt.exe
+
+bin:
+  kulala-fmt: "{{source.asset.file}}"
```

It seems like the package started getting published to npm starting at v2?

```suggestion
  id: pkg:npm/%40mistweaverco/kulala-fmt@2.2.0

bin:
  kulala-fmt: npm:kulala-fmt
```

---

That's true. Did not think this would go through anymore.

---

Understandable 😅